testplan/common/entity/base.py

```diff
@@ -315,7 +315,22 @@ class Entity(logger.Loggable):
     CONFIG = EntityConfig
     STATUS = EntityStatus
 
-    def __init__(self, **options):
+    def __init__(self,
```

Comment:
I don't think Entity is considered 'public API class/function', so it doesn't need to be changed.

Reply:
No definitely not. Don't worry about converting abstract bass classes like Entity, Resource etc. Only the ones that testplan users will be importing and initialising directly.